package.json

```diff
@@ -14,30 +14,31 @@
   "license": "MIT",
   "main": "index.js",
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=6.9.0"
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "@babel/core": "~7.21.3",
```

Comment:
Also it's probably a good time to move dev dependencies into `devDependencies` section.